app/components/annotation/Answer.js

```diff
@@ -1,3 +1,4 @@
+// TODO: marked for deletion: topic picking view doesn't need Q&A
```

Comment:
ok @nickbadams if this really is changing that drastically then new UX will be critical. otherwise this will take much longer to achieve.